### PR TITLE
Feature/social images

### PR DESCRIPTION
**Pull request by noahmanger**
Adds custom social images for the different sections of the site, with fallback to a generic one. 

To do this, I had to:

- Make a function for `CustomPage`, `ResourcePage` and `CollectionPage` instances to figure out which section they're in, since they can be used in multiple places.
- Make template filters for generating static paths based on the parent section

It works, but let me know if there's a better way to do this.

Requires https://github.com/18F/fec-style/pull/699

### CODE REVIEW

File: fec/home/models.py [hunk 1]
```diff
@@ -168,10 +187,6 @@ def content_section(self):
         return ''
 
 
-class LandingPage(ContentPage):
-    template = 'home/registration-and-reporting/landing_page.html'
```
Comment (ccostino) [inline fec/home/models.py]: Ah, we're getting rid of this model?

File: fec/home/models.py [hunk 2]
```diff
@@ -54,6 +54,25 @@
...
+    }
+    ancestors = page.get_ancestors()
+    content_sections = [
+        slugs.get(ancestor.slug) for ancestor in ancestors
+        if slugs.get(ancestor.slug) != None
+    ]
+    if len(content_sections):
+        return content_sections[0]
+    else:
+        return ''
```
Comment (ccostino) [inline fec/home/models.py]: Nice!

File: fec/home/templatetags/filters.py [hunk 3]
```diff
@@ -87,3 +88,25 @@ def splitlines(value):
...
+        return static('img/social/general.png')
+
+
+@register.filter(name='get_touch_icon')
+def get_touch_icon(content_section, dimension):
+    """
+    Returns a path to a touch icon for the given dimension and content_section
+    """
+    print('====')
+    print(content_section)
```
Comment (ccostino) [inline fec/home/templatetags/filters.py]: Do we need these `print` statements in here?

File: fec/home/models.py (hunk 1 above)
Comment (noahmanger) [inline fec/home/models.py]: Yep! We don't need it anymore because that section doesn't exist.

File: fec/home/templatetags/filters.py (hunk 3 above)
Comment (noahmanger) [inline fec/home/templatetags/filters.py]: Oops! Fixed.

**Comment by noahmanger** [comment]
Added the fec-style bump here.

**Pull request merged**